README.md

```diff
@@ -26,17 +27,14 @@
 
 > For performance reasons it is recommended that you use the HTTP client if you are building an application.
 
-## Quick start
-
-Learn the basics by setting up and interacting with the Ceramic CLI. [Quick start guide →](https://developers.ceramic.network/build/quick-start/)
 
 ## Installation and usage
 
-Full documentation on installation and usage can be found on the [Ceramic documentation site →](https://developers.ceramic.network/build/installation/)
```

Comment:
do we still have a getting started guide of some kind we can link here for how developers should start learning and getting set up to build?

Reply:
Yeah, that's the main problem. I don't want to send devs to the old quickstart (https://developers.ceramic.network/build/cli/quick-start/) because it's very outdated and is still using glaze. The only alternative we have is a getting started guide on composedb docs which I am afraid might be a bit too confusing for devs (as in, why are they being sent to composedb documentation). However, we can use it as an interim steps until we make changes to the overall docs structure and update outdated content.